blib2to3/pgen2/driver.py

```diff
@@ -25,17 +25,39 @@
 
 # Pgen imports
 from . import grammar, parse, token, tokenize, pgen
+from typing import (
+    Any,
+    Callable,
+    IO,
+    Iterable,
+    List,
+    Optional,
+    Text,
```

Comment:
You can just use str since this is Python 3-only code.

Reply:
I'll leave this to a follow-up, since the lib2to3 stubs used Text pretty heavily